### ISSUE TITLE
Could you share your dotfiles?

### ISSUE THREAD

**Issue by UltiRequiem** [ms-jpq/coq_nvim#64]
It would be useful to be guided when configuring this plugin.

Btw very good work, I really love this plugin, it has made me change almost all my configuration :D

**Comment by ayamir** [comment]
I need it too!

**Comment by jdhao** [comment]
+1 We would like to see a fast as fuck neovim config, LOL 😅

**Comment by ms-jpq** [comment]
my `.vimrc` is actually slow, its written in python which has a startup cost that will make all optimizations useless.

its also 4000 lines long so yeah have fun:

https://github.com/cyproterone/nvim2/tree/nvim

**Comment by ms-jpq** [comment]
by slow i meant like `0.4` seconds, not vscode slow

**Comment by ms-jpq** [comment]
It actually doesnt have too many plugins. I wrote most things myself so I removed most third party dependencies.

The major ones I use are the nvim official ones, my own, a theme, `fzf`, and `vim-easy-align`, `vim-unimpaired`, `vim-surround`, `vim-repeat`.

the rest i can live without, but yeah, im a huge fan of [junegunn](https://github.com/junegunn) and [tpope](https://github.com/tpope).

I think their stuff is really good and not really worth rewriting it myself, except the indentation detection one, I think my modulo based algorithm makes more sense, and the commenting one where I think I wrote a fairly competitive alternative.

---
